src/app/fdctl/configure/ethtool.c

```diff
@@ -157,6 +191,23 @@ check_device( const char * device,
     }
   }
 
+  /* check generic-receive-offload, which is entirely incompatible with
+   * AF_XDP and QUIC */
+
+  /* command for getting generic-receive-offload */
+  struct ethtool_value gro = { .cmd = ETHTOOL_GGRO, .data = 0 };
+
+  /* attach command to ifr */
+  ifr.ifr_data = (void *)&gro;
+
+  /* execute command */
+  if( FD_UNLIKELY( ioctl( sock, SIOCETHTOOL, &ifr ) ) ) {
+    if( FD_LIKELY( errno != EOPNOTSUPP ) ) {
```

Comment:
EOPNOTSUPP here means the device doesn't support GRO so it must be disabled?

Reply:
I'm pretty sure that's what it means. The alternative is that it's unreadable, and yet enabled. This seems extremely unlikely